### PR TITLE
fixup labels

### PR DESCRIPTION
**Pull request by softprops**
some followup to https://github.com/softprops/hubcaps/pull/121

### CODE REVIEW

File: src/issues/mod.rs [hunk 1]
```diff
@@ -103,15 +102,14 @@ impl<C: Clone + Connect> IssueLabels<C> {
...
-            which: which,
             number: number,
         }
     }
 
     fn path(&self, more: &str) -> String {
         format!(
             "/repos/{}/{}/{}/{}/labels{}",
-            self.owner, self.repo, self.which, self.number, more
+            self.owner, self.repo, "issues", self.number, more
```
Comment (dbaron) [inline src/issues/mod.rs]: I suppose this could probably be reverted to the way it was before #121, with the issues inside of the first string.

Comment (softprops) [reply src/issues/mod.rs]: nice catch! Ill fix that merge and publish this weekend

**Comment by softprops** [comment]
@dbaron published 0.4.6 which should now have interfaces for pulling back pr labels and updating/removing them. Thanks for the nudge on this I can thing of some places were tooling to manage these would help where I work

**Comment by dbaron** [comment]
Thanks.  I've made the change to my bot that uses it, and tested that 0.4.6 works with my bot (see the change linked from dbaron/wgmeeting-github-ircbot#26).

**Pull request merged**